CONTRIBUTING.md

```diff
@@ -13,14 +13,98 @@ Check out the [Projects](https://github.com/jMonkeyEngine/jmonkeyengine/projects
 
 When you're ready to submit your code, just make a [pull request](https://help.github.com/articles/using-pull-requests).
 
-- Do not commit your code until you have received proper feedback.
 - In your commit log message, please refer back to the originating forum thread (example) for a ‘full circle’ reference. Also please [reference related issues](https://help.github.com/articles/closing-issues-via-commit-messages) by typing the issue hashtag.
 - When committing, always be sure to run an update before you commit. If there is a conflict between the latest revision and your patch after the update, then it is your responsibility to track down the update that caused the conflict and determine the issue (and fix it). In the case where the breaking commit has no thread linked (and one cannot be found in the forum), then the contributor should contact an administrator and wait for feedback before committing.
```

Comment:
> - When committing, always be sure to run an update before you commit.

I'm finding the language here a little vague/misuse of the term 'commit'. I know it predates this PR, but since we're discussing the file, maybe:
> - When starting a new project, always be sure to pull the latest revision of the jMonkeyEngine repository master branch before creating your new project branch.

or similar?

Reply:
There are many instructions in this file that irritate me. I think they should be addressed in a separate PR.